modules/ROOT/pages/managing-cloudhub-specific-settings.adoc

```diff
@@ -44,7 +44,7 @@ image::subscriptionlevel.png[SubscriptionLevel]
 [NOTE]
 --
 *The Number of Workers Is Not the Same as the Number of Applications* +
```

Comment:
We should not be using boldface and title-style capitalization for emphasis.
The fact that this is isolated as a Note is emphasis enough.
In general, use context rather than formatting to provide emphasis. When absolutely required, use italics.